api/proto/certs/certs.proto

```diff
@@ -32,23 +21,15 @@ enum certHashAlgorithm {
     HASH_NONE = 0;
     HASH_SHA256_16bytes = 1;  // hash with sha256, the 1st 16 bytes of result in 'certHash'
 }
+
 message ZCert {
    certHashAlgorithm hashAlgo = 1; //hash method used to arrive at certHash
    bytes certHash = 2;             //Hash of the cert, computed using hashAlgo
    ZCertType type = 3;             //what kind of certificate(to identify the target use case)
    bytes cert = 4;                 //X509 cert in .PEM format
-   ZCertProperties properties = 5; //properties of this certificate
 }
 
 enum ZCertType {
```

Comment:
As been discussed elsewhere, we need an Intermediate cert type as well.

Reply:
Added CERT_TYPE_CONTROLLER_INTERMEDIATE